python/pyspark/sql/functions.py

```diff
@@ -188,11 +195,14 @@ def _():
     globals()[_name] = since(1.4)(_create_binary_mathfunction(_name, _doc))
 for _name, _doc in _window_functions.items():
     globals()[_name] = since(1.4)(_create_window_function(_name, _doc))
+for _name, _doc in _functions_1_5.items():
```

Comment:
i don't think u need this, since you explicitly defined it. you might need to add it to __all__ at the top of the file